devel/libuv/files/patch-libuv-1-23-2-tiger.diff

```diff
@@ -1,56 +1,24 @@
-diff --git src/unix/fs.c src/unix/fs.c
-index 3db5f89..3f0d719 100644
---- src/unix/fs.c
-+++ src/unix/fs.c
-@@ -59,7 +59,7 @@
- # include <sys/sendfile.h>
- #endif
- 
--#if defined(__APPLE__)
-+#if ( defined(__APPLE__) && MAC_OS_X_VERSION_MAX_ALLOWED >= 1050)
- # include <copyfile.h>
- #elif defined(__linux__) && !defined(FICLONE)
- # include <sys/ioctl.h>
-@@ -589,7 +589,7 @@ static ssize_t uv__fs_sendfile(uv_fs_t* req) {
+diff -u src/unix/fs.c.orig src/unix/fs.c
+--- src/unix/fs.c.orig   2019-05-04 14:35:17.000000000 -0700
++++ src/unix/fs.c        2019-05-04 14:40:00.000000000 -0700
+@@ -729,7 +729,7 @@
  
      return -1;
    }
 -#elif defined(__APPLE__)           || \
-+#elif defined(__APPLE__) && MAC_OS_X_VERSION_MAX_ALLOWED >= 1050  || \
++#elif defined(__APPLE__) && MAC_OS_X_VERSION_MAX_ALLOWED >= 1050           || \
```

Comment:
I think this whitespace isn't necessary ... yes?

Reply:
Also probably want this clause in () because they go together outside the ||s.

Reply:
done